### PR TITLE
HDDS-5983. Prefix Parser tool should only work for FSO buckets.

### PR DESCRIPTION
**Pull request by mukul1987**
## What changes were proposed in this pull request?
Prefix Parser tool will only work against FSO buckets, this jira adds a check for that

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-5983

## How was this patch tested?
Existing tests

### CODE REVIEW

**Review by bharatviswa504 — APPROVED** [review]
+1 LGTM

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/PrefixParser.java [hunk 1]
```diff
@@ -148,6 +148,14 @@ public void parse(String vol, String buck, String db,
       return;
     }
 
+    BucketLayout bucketLayout = info.getBucketLayout();
+    if (!bucketLayout.isFileSystemOptimized()) {
+      System.out.println("Prefix tool only works for FileSystem Optimized" +
+              "bucket. Bucket Type is:" + bucketLayout);
```
Comment (rakeshadr) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/PrefixParser.java:154]: @mukul1987 can you please modify message `"Bucket Type is:"` -> to -> `"Bucket Layout is:"`

**Review by rakeshadr — COMMENTED** [review]
Overall +1 LGTM. I've added just a minor comment.

**Review by rakeshadr — APPROVED** [review]
+1 LGTM, thanks @mukul1987

**Pull request merged**